src/init.cpp

```diff
@@ -964,17 +964,29 @@ bool AppInitParameterInteraction()
 
     // also see: InitParameterInteraction()
 
-    // Warn if network-specific options (-addnode, -connect, etc) are
+    // Error if network-specific options (-addnode, -connect, etc) are
     // specified in default section of config file, but not overridden
     // on the command line or in this network's section of the config file.
     std::string network = gArgs.GetChainName();
-    for (const auto& arg : gArgs.GetUnsuitableSectionOnlyArgs()) {
```

Comment:
I don't think this line needs to change. Below you can simply check for `errors.empty()`, which should do the exactly same.

Reply:
`class "bilingual_str" has no member "empty"`. 

Would it be reasonable to use `errors.translated.empty()` for this purpose?

Reply:
> `class "bilingual_str" has no member "empty"`.
> 
> Would it be reasonable to use `errors.translated.empty()` for this purpose?

Feel free to get it from https://github.com/bitcoin/bitcoin/pull/18927/commits/1df513531f04ef31d55d2fa2acc146d526745788 :)

Reply:
Oh, is that not yet merged. Sorry didn't realize